src/index.js

```diff
@@ -0,0 +1,177 @@
+
+/**********************1*********************/
+
+
+
+
+const data = [
+  { value: "1", label: "One" },
+  { value: "2", label: "Two" },
+  { value: "3", label: "Three" },
+];
+
+const values = data.reduce((values, { value }) => {
```

Comment:
why reduce an array to another array ?
if you're looking to transform an array while keeping the type use the array prototype map function:

const values = data.map(item => item.value);